### PR TITLE
Grantee history

### PR DESCRIPTION
**Pull request by janpaul123**
See individual commits.

Tested in Chrome (OS X), Firefox (OS X), Safari (OS X), IE11 (Windows), and various random mobile browsers through Browserstack.

Includes grantees overview on home page.

Does require updating _config.yml with your own copy, plus maybe some link tweaks here and there (do we have videos for all grantees?).

Grantees page:
[Desktop](https://cloud.githubusercontent.com/assets/177461/26188165/870ec5ba-3b52-11e7-9156-43eca6aa48da.png)
[Mobile](https://cloud.githubusercontent.com/assets/177461/26188169/8978c36e-3b52-11e7-9bf8-5ec4b044ef0e.png)

Home page:
[Desktop](https://cloud.githubusercontent.com/assets/177461/26188179/95d07b8e-3b52-11e7-95ea-9ae923438b80.png)
[Mobile](https://cloud.githubusercontent.com/assets/177461/26188183/97530b2a-3b52-11e7-973b-cf0b6150d961.png)

PS: Do check out the CSS. Stuff's pretty crazy here and there. 😸

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -45,11 +47,48 @@ current_org:
...
+    amount: $5,345
+    focus: Refugee Support
+    contributors: 87
+    slug: thrive
+  - month: January
+    year: 2017
+    name: Making Contact Radio
+    short_name: Making Contact Radio # On home page
+    description: Founded in 1994, Making Contact is an independent non-profit organization committed to investigative journalism, in-depth critical analysis, the promotion of civic participation and the dissemination of educational material. What began as a 13-week experiment in highlighting progressive community perspectives has become an enduring platform for underrepresented groups and community voices.
+    video: '???'
```
Comment (janpaul123) [inline _config.yml]: This def. needs updating.

File: _config.yml [hunk 2]
```diff
@@ -45,11 +47,48 @@ current_org:
   body: "The mission of Coal River Mountain Watch is to stop the destruction of our communities and environment by mountaintop removal mining, to improve the quality of life i…"
   url: https://blog.vocalminority.us/vocal-minority-funds-coal-river-mountain-watchs-fight-for-environmental-justice-41cdb74b5f6d
 
+grantees:
```
Comment (janpaul123) [inline _config.yml]: You guys need to add your own copy here!

**Comment by kball** [comment]
Looks good to me! I did one copy tweak and added video link for Making Contact... @nuprin do you have any copy updates you want to make?

**Comment by janpaul123** [comment]
😄 😸 😄 😸 😄 😸 😄!!!!

**Pull request merged**